e2e/rest-client-e2e/tests/rest-client.spec.ts

```diff
@@ -47,54 +49,103 @@ describe('rest-client e2e', () => {
     });
 
     describe('client-endpoint', () => {
-        const endpoint = uniq('test-endpoint');
+        const libName = uniq('test-endpoint');
+        const endpointsDir = uniq('endpoints');
+        const httpclientName = uniq('http-client');
+
+        beforeAll(async () => {
+            await runNxCommandAsync(
+                `generate @ensono-stacks/rest-client:http-client ${httpclientName}`,
+            );
+        });
 
         it('should create lib in the specified directory', async () => {
             await runNxCommand(
-                `generate @ensono-stacks/rest-client:client-endpoint ${endpoint} --methods=get,post --directory=endpoints --httpClient="@proj/http-client" --no-interactive`,
+                `generate @ensono-stacks/rest-client:client-endpoint ${libName} --methods=get,post,patch,put,delete,head,options --directory=${endpointsDir} --httpClient="@proj/${httpclientName}" --no-interactive`,
             );
 
             expect(() =>
                 checkFilesExist(
-                    `libs/endpoints/${endpoint}/v1/src/index.ts`,
-                    `libs/endpoints/${endpoint}/v1/src/index.test.ts`,
-                    `libs/endpoints/${endpoint}/v1/src/index.types.ts`,
-                    `libs/endpoints/${endpoint}/v1/project.json`,
-                    `libs/endpoints/${endpoint}/v1/tsconfig.json`,
+                    `libs/${endpointsDir}/${libName}/v1/src/index.ts`,
+                    `libs/${endpointsDir}/${libName}/v1/src/index.test.ts`,
+                    `libs/${endpointsDir}/${libName}/v1/src/index.types.ts`,
+                    `libs/${endpointsDir}/${libName}/v1/project.json`,
+                    `libs/${endpointsDir}/${libName}/v1/tsconfig.json`,
+                    `.env.local`
                 ),
             ).not.toThrow();
 
-            const expectedImportName = `@proj/endpoints/${endpoint}/v1`;
+            const expectedImportName = `@proj/${endpointsDir}/${libName}/v1`;
 
             const tsConfig = readJson('tsconfig.base.json');
             expect(tsConfig.compilerOptions.paths).toHaveProperty(
                 expectedImportName,
-                [`libs/endpoints/${endpoint}/v1/src/index.ts`],
+                [`libs/${endpointsDir}/${libName}/v1/src/index.ts`],
             );
-        }, 120000);
+        });
+
+        it('should build the library without failure', async () => {
+            const result = await runNxCommandAsync(
+                `build ${endpointsDir}-${libName}-v1`,
+            );
+
+            expect(result.stderr).not.toEqual(expect.stringContaining('failed'));
```

Comment:
Ideally swap this out to use the runTarget function so that we're consistent

Reply:
Using runTarget now. Same applies to the other areas.